lib/tasks/example_import.rake

```diff
@@ -0,0 +1,52 @@
+require 'csv'
+require './app/models/customer'
+require './app/models/application_record'
```

Comment:
did requiring this get around the uninitialized constant error?

Reply:
Those did nothing so we axed them. The fix was adding the correct environment at to each method so that it knows what parts of rails to load before executing. Molly's mentor was nice enough to help us out with it. In some environments you don't want the rake's to be available.

Reply:
Can we go over this part in person? I think this makes sense to me, but I just want to double check that I'm thinking about the same thing as you.